Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -57,7 +57,7 @@ class SuccOrder (α : Type*) [Preorder α] where
   le_succ : ∀ a, a ≤ succ a
   /-- Proof of interaction between `succ` and maximal element-/
   max_of_succ_le {a} : succ a ≤ a → IsMax a
-  /-- Proof that `succ` satisfies ordering invariants between `LT` and `LE`-/
+  /-- Proof that `succ a` is the minimal element greater than `a`-/
```

Comment:
Is "minimal" the correct word? Isn't the successor in a partial order always unique, meaning we can replace this by "minimum"?

Reply:
This definition only assumes a preorder, not a partial order

Reply:
In a preorder, all minimal elements grater than `a` should be equivalent, I believe? So my statement is still approximately right.

Really, successor orders on preorders are somewhat silly. I don't think we lose anything by defining them for preorders like this, but we should really be thinking about them as either partial or total orders.

Reply:
Informally, "minimum element" is not something we really say, at least much less than we would say "minimal element", so I'm happy with the docstring as it is

Reply:
Sorry, "least" element would be the proper wording.

Can we at least still say that it is unique in the case of partial orders?

Reply:
I changed it to least.